### ISSUE TITLE
ONNXConversion silently ignores an exception

### ISSUE THREAD

**Issue by ArtemJDS** [do-mpc/do-mpc#456]
I added an unimplemented function to a model and passed it to ONNXConversion. **It worked without exception or warning** about lack of the function realization.
See details below.

I'm working in google colab, python **3.10.12.**
I have downloaded casady and do-mpc and in general followed the example provided here <https://www.do-mpc.com/en/latest/api/do_mpc.sysid.ONNXConversion.html>, but for the part of defining last activation as 'softmax'.

`

    pip install casadi
    pip install do-mpc[full];
    pip install tf2onnx
    
    from casadi import *
    import do_mpc
    from do_mpc.sysid import ONNXConversion as Converter
    import tf2onnx
    import keras
    
    model = keras.Sequential([keras.Input(3), 
                              keras.layers.Dense(1000, activation = 'relu'), 
                              keras.layers.Dense(3, activation = 'softmax')])
`

Then I converted the model as in the example above

`

    model_input_signature = [
        tf.TensorSpec(np.array((1, 3)), name='input'),
    ]
    output_path = os.path.join('models', 'model.onnx')
    
    onnx_model, _ = tf2onnx.convert.from_keras(model,
        output_path=output_path,
        input_signature=model_input_signature
    )
    
    casadi_converter = do_mpc.sysid.ONNXConversion(onnx_model)
    print(casadi_converter)
`

So was the output
`

      ONNX2Casadi model 'casadi_model' 
      ----------------------------------
      Call 'convert' by supplying the inputs with respective name and shape below.
      Input shape of 'input' is (1, 3)
      ----------------------------------
      Query the instance with the following keywords to obtain the CasADi expression of the respective layer or graph operation node:
       - 'input'
       - 'sequential_4/dense_8/MatMul:0'
       - 'sequential_4/dense_8/Relu:0'
       - 'sequential_4/dense_9/MatMul:0'
       - 'dense_9'
`
As you see, there is no softmax function (there must be none, it is not implemented). But I received **no signal from the programm** about that.

It seems there must be an exception as convert method implements it.

`    
        
              if hasattr(self.operations, n.op_type):
                  out = getattr(self.operations, n.op_type)(*ins, attribute=n.attribute)
              else:
                  raise Exception("Operation '{}' not implemented. Please consider the limited set of operations available to the tool.".format(n.op_type))

`

**Comment by ArtemJDS** [comment]
My bad, I haven't converted yet
